lib/dspace-item.ts

```diff
@@ -0,0 +1,60 @@
+import { Bitstream, Metadata } from './dspace-schemas';
+
+/**
+ * Returns an array of metadata items that match the given key.
+ * @param metadataItems - List of metadata to search.
+ * @param key - The key for the metadata items to find.
+ *
+ * @returns An array with metadata items for the given key.
+ * If no metadata items match the key an empty array is returned.
+ */
+export const findMetadataItems = (
+  metadataItems: Metadata[],
+  key: string
+): Metadata[] => metadataItems.filter((m) => m.key === key);
```

Comment:
Consider using [lodash's filter function](https://lodash.com/docs/4.17.15#filter), which supports a nice shorthand for a case like this:

```ts
=> _.filter(metadataItems, { key });
```